setup.py

```diff
@@ -48,6 +48,7 @@ def package_files(directory, strip_leading):
           "simple_pid",
           'progress',
           'typing_extensions',
+          'psutil'
```

Comment:
Because this only runs on the car, it might go into the `pi` section.

Reply:
Can you move this into the `'pi'` section?

Reply:
I don't understand why should it be limited to `pi` or `nano`? What if I want. to verify performance running in simulation?
Sorry for a delayed response.

Reply:
Fair point - then we should add it to the conda.yml files too.

Reply:
@DocGarbanzo - should I add it to `envs/mac.yml` and `envs/ubuntu.yml` ?

Reply:
Yeah, I think the second one got renamed to 'pc.yml' in the mean time.

Reply:
fixed